### ISSUE TITLE
episode 03 - clarity suggestion to text

### ISSUE THREAD

**Issue by alp5088** [datacarpentry/python-ecology-lesson#514]
As I was preparing my teaching demonstration for Carpentries instructor training I was working through the episodes in the Data Analysis and Visualization in Python for Ecologists lesson. I am relatively new to Python/programming in general but would suggest adding a sentence or two to the descriptions of loc and iloc in the "Slicing Subsets of Rows and Columns in Python" episode (https://datacarpentry.org/python-ecology-lesson/03-index-slice-subset/#slicing-subsets-of-rows-and-columns-in-python). For a novice like myself, I was confused about the definitions/explanations given for loc and iloc. My quick suggestion is to add more descriptive language to this area. Something like "iloc indexing selects rows and/or columns at specific integer positions" and "loc indexing selects rows and/or columns at specific labels".. or something similar. Hope I have that right! :)

**Comment by btovar** [comment]
@alp5088 Yes, thanks, this is a great suggestion. iloc and loc are always a little confusing.
